### PR TITLE
refactor(external_cmd_selector): rework parameters

### PR DESCRIPTION
**Pull request by batuhanbeytekin**
## Description
Implement the ROS Node configuration layout described in https://github.com/orgs/autowarefoundation/discussions/3371

## Tests performed
Package built and launch locally.

## Effects on system behavior
More reliable and faster parameter configuration file creation.

## Pre-review checklist for the PR author
The PR author **must** check the checkboxes below when creating the PR.
- [x] I've confirmed the [contribution guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/).
- [x] The PR follows the [pull request guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/).

## In-review checklist for the PR reviewers
The PR reviewers **must** check the checkboxes below before approval.
- [ ] The PR follows the [pull request guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/).
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author
The PR author **must** check the checkboxes below before merging.
- [x] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

**Review by mitsudome-r — APPROVED** [review]

File: control/autoware_external_cmd_selector/README.md [hunk 1]
```diff
@@ -32,3 +32,7 @@ The current mode is set via service, `remote` is remotely operated, `local` is t
 | `/external/selected/hazard_lights_cmd`                 | autoware_vehicle_msgs::msg::HazardLightsCommand   | Pass through hazard light with current mode.    |
 | `/external/selected/heartbeat`                         | TBD                                               | Pass through heartbeat with current mode.       |
 | `/external/selected/turn_indicators_cmd`               | autoware_vehicle_msgs::msg::TurnIndicatorsCommand | Pass through turn indicator with current mode.  |
+
+## Parameters
+
+{{json_to_markdown("/control/autoware_external_cmd_selector/schema/external_cmd_selector.schema.json")}}
```
Comment (mitsudome-r) [inline control/autoware_external_cmd_selector/README.md:38]: The path for this function should be the relative path from the root directory so please 

```suggestion
{{json_to_markdown("control/autoware_external_cmd_selector/schema/external_cmd_selector.schema.json")}}
```

Comment (mitsudome-r) [reply control/autoware_external_cmd_selector/README.md:38]: @batuhanbeytekin It seems like you resolved the conversation, but it seems like the issue exists. Could you remove the first `/` in the path?

**Comment by batuhanbeytekin** [comment]
Hi @TakaHoribe, @rej55, @satoshi-ota, @shmpwk, @taikitanaka3, @takayuki5168, @tkimura4,

One approval is needed to merge this PR. Could you please review it?

**Review by shmpwk — APPROVED** [review]
LGTM

**Pull request merged**